checkra1n-downloader/checkra1n-downloader.sh

```diff
@@ -16,10 +16,13 @@ arch=$(uname -m)
 
 if [ "$(echo "$links" | grep "$arch")" ]; then
   link=$(echo "$links" | grep "$arch")
+else
+if arch=aarch64 ; then
+arch=arm64
```

Comment:
It works but I would prefer the whole script written in the same way, this might look better and more readable:
```shell
if [ "$arch" = "aarch64" ]; then
  arch="arm64"
```